### DISCUSSION TITLE
Entitlement Management: Access Packages

### DISCUSSION THREAD

**Discussion by dmaloney02** [microsoftgraph/msgraph-sdk-powershell#2805] [Q&A]
Has anyone had any luck created access packages as assigned catalog items to the access package?

I've added a security group to the catalog via New-MgEntitlementManagementResourceRequest

The resulting object seems to be missing a role property.  Graph API documentation states these roles should be automatically present:

In [Microsoft Entra entitlement management](https://learn.microsoft.com/en-us/graph/api/resources/entitlementmanagement-overview?view=graph-rest-1.0), an access package resource role is a reference to a role defined in a resource. These roles are automatically present after a resource is added to an access package catalog. A group can have two roles, one for the owner and another for the member.

I can't use the New-MgEntitlementManagementAccessPackageResourceRoleScope command to add the Security group to the access package because the role information is required.

**Comment by mlibenocg** [comment]
Check out Example 3: Add group membership to an access package at https://learn.microsoft.com/en-us/powershell/module/microsoft.graph.identity.governance/new-mgentitlementmanagementaccesspackageresourcerolescope?view=graph-powershell-1.0 .
